content/overview/pricing/rates.md

```diff
@@ -78,6 +78,16 @@ Free of charge and without any paperwork, anyone with a government email address
 
 [Learn more about sandboxes.]({{< relref "overview/pricing/free-limited-sandbox.md" >}})
 
+## Quota pricing
+
+Quotas are priced at $0.0033 per MB reserved per day and assessed monthly. In practice, this works out to $99 per GB per month, based on a 30-day month:
+
+`$0.0033 * 1000 MB * 30 days = $99 per GB per month`
+
+The monthly cost will vary slightly from $92.40 per GB in a 28-day month to $102.30 per GB in a 31-day month.
```

Comment:
The issue we're really parsing here is whether we can say "$99 per GB per month" vs. "$0.0033 per MB per day". The former is easier to comprehend, but the latter is more accurate. What I want to do is lead with the former but provide the latter so people understand the technicality and/or don't accidentally misrepresent the price.

So the question is: is it easier to just provide the $0.0033 figure? Or do we accept the risk of uncertainty from just providing the $99 figure? That's where we started out here, but in conversation, @brittag I were discussing that it isn't really correct so it might be wrong to say it that way. 

It's also technically more like $100.38 per GB per month. $99 is a great estimate for understanding how it looks over the course of 30 days, but there are still 5 extra days in the year after that. You consider this stuff long enough and leaving it at $99 starts to feel insufficient. But again, maybe we're way out in front of our customers.